pyccel/codegen/printing/cwrappercode.py

```diff
@@ -405,9 +410,10 @@ def _body_management(self, used_names, variable, collect_var, check_type = False
         used_names : list of strings
             List of variable and function names to avoid name collisions
         Variable : Variable
```

Comment:
The first `Variable` should not be capitalized

Reply:
Fixed 7364ac9f